compiler/luci/logex/src/FormattedGraph.cpp

```diff
@@ -2255,6 +2257,11 @@ bool NodeSummaryBuilder::build(const loco::Node *node, locop::NodeSummary &s) co
   BUILD_GRP(CIRC);
   BUILD_GRP(VIRT);
 
+  if (CircleNodeSummaryBuilder().build(node, _tbl, s))
```

Comment:
Is it impossible to remove `BUILD_GRP` codes in this PR?

Reply:
`CircleNodeSummaryBuilder` is empty for now :)
So during implementation of `CircleNodeSummaryBuilder` for each operation, `BUILD_GRP` will be removed sequentially :D